src/components/fields/ColorwayPicker.js

```diff
@@ -0,0 +1,31 @@
+import ColorscalePickerWidget from '../widgets/ColorscalePicker';
```

Comment:
yup, that's a really nice one to expose :)

Reply:
Yeah :)

What do you think of the various renamings? I did what I could to make them clearer... we used to import ColorscalePicker from Colorscale.js which has bugged me for months ;)

Reply:
yup, the names are good and clear to me!
great job on separating them all :)